packages/docusaurus-plugin-content-docs/src/globalData.ts

```diff
@@ -12,6 +12,7 @@ export function toGlobalDataDoc(doc: DocMetadata): GlobalDoc {
     id: doc.unversionedId,
     path: doc.permalink,
     sidebar: doc.sidebar,
+    ...(doc.frontMatter && {frontMatter: doc.frontMatter}),
```

Comment:
I think we don't want frontmatter here, as it increases the size of all the site's pages